pkg/storaged/stratis/filesystem.jsx

```diff
@@ -183,15 +183,9 @@ export function make_stratis_filesystem_page(parent, pool, fsys,
         });
     }
 
-    let mp_text;
-    if (mount_point) {
-        mp_text = client.strip_mount_point_prefix(mount_point);
-        if (mp_text == false)
-            return;
-        if (!fs_is_mounted)
-            mp_text = mp_text + " " + _("(not mounted)");
-    } else
-        mp_text = _("(not mounted)");
+    const mp_text = mount_point_text(mount_point, fs_is_mounted);
+    if (mp_text == null)
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19824-20240108-104540-fcdf3177-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)